pause.go

```diff
@@ -9,18 +9,36 @@ package main
 /*
 #cgo CFLAGS: -Wall
 #define _GNU_SOURCE
-#include <signal.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
 #include <unistd.h>
+#include <errno.h>
 
 #define PAUSE_BIN "pause-bin"
 
-void __attribute__((constructor)) sandbox_pause(int argc, const char **argv) {
-	if (argc != 2 || strcmp(argv[1], PAUSE_BIN)) {
-		return;
+void __attribute__((constructor)) sandbox_pause(void) {
+	FILE *f;
+	int len, do_pause = 0;
+	size_t n = 0;
+	char *p = NULL;
+
+	f = fopen("/proc/self/cmdline", "r");
+	if (f == NULL) {
+		perror("failed to open proc");
+		exit(-errno);
+	}
+	while ((len = getdelim(&p, &n, 0, f)) != -1) {
+		if (len == strlen(PAUSE_BIN) + 1 && strcmp(p, PAUSE_BIN)) {
+			do_pause = 1;
+			break;
+		}
 	}
+	fclose(f);
+	free(p);
+
+	if (do_pause == 0)
```

Comment:
please use braces

Reply:
Not really necessary right? C code doesn't need to follow go coding style here.